paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -63,7 +62,6 @@ internal class DefaultFlowController @Inject internal constructor(
     private val lifecycleScope: CoroutineScope,
     lifecycleOwner: LifecycleOwner,
     private val statusBarColor: () -> Int?,
-    private val authHostSupplier: () -> AuthActivityStarterHost,
```

Comment:
since we no longer need `authHostSupplier`, it could be removed from dagger graph.
You can delete [FlowControllerComponent.Builder.authHostSupplier](https://github.com/stripe/stripe-android/blob/54276bcccbce9f027a9a7d6eb9d72d666318b7eb/paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/FlowControllerComponent.kt#L56),
then delete it from the [factory method](https://github.com/stripe/stripe-android/blob/e34a3be618f57629dc4b9154b81a642454bd4275/paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt#L435), 
then delete it from the [caller](https://github.com/stripe/stripe-android/blob/54276bcccbce9f027a9a7d6eb9d72d666318b7eb/paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerFactory.kt#L22) of that factory method

Reply:
done here: https://github.com/stripe/stripe-android/pull/4175/commits/ffdb7d5e7a89321cc26c928952ff45ad60357909